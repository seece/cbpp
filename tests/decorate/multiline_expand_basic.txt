#macro TESTMACRO(a, b)
Log(s:"a is ", d:a);
Log(s:"b is ", d:b);
#endmacro

TESTMACRO(1,2)
TESTMACRO(3,4)