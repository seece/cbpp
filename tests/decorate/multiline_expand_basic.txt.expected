// #macro TESTMACRO(a, b)
// Log(s:"a is ", d:a);
// Log(s:"b is ", d:b);
// #endmacro

Log(s:"a is ", d:1);
Log(s:"b is ", d:2);
Log(s:"a is ", d:3);
Log(s:"b is ", d:4);